doc/history.rst

```diff
@@ -316,8 +316,8 @@ Version 0.11
 
   - If you have the `keyring <https://github.com/jaraco/keyring>`_ package
     installed, flit can use it to store your password, rather than keeping it
-    in plain text in ``~/.pypirc``.
-  - If ``~/.pypirc`` does not already exist, and you are prompted for your
+    in plain text in the .pypirc file.
+  - If the .pypirc file does not already exist, and you are prompted for your
```

Comment:
I might ask you to revert this bit - it's not normally necessary to modify release notes from several years ago, unless there's a broken link or something.

Reply:
Sure! Sorry, I thought this was documentation. Will revert.

Reply:
No problem! It is part of the docs, but it's specifically saying what changed from one release to the next, and this bit is about the 0.11 release a few years ago.